target_chains/solana/sdk/js/pyth_solana_receiver/src/PythSolanaReceiver.ts

```diff
@@ -93,6 +100,18 @@ export class PythTransactionBuilder extends TransactionBuilder {
    * Add instructions to post price updates to the builder.
    *
    * @param priceUpdateDataArray the output of the `@pythnetwork/price-service-client`'s `PriceServiceConnection.getLatestVaas`. This is an array of verifiable price updates.
+   *
+   * @example
+   * ```typescript
+   * const priceUpdateData = await priceServiceConnection.getLatestVaas([
+   *    SOL_PRICE_FEED_ID,
+   *    ETH_PRICE_FEED_ID,
+   * ]);
+   *
+   * const transactionBuilder = pythSolanaReceiver.newTransactionBuilder({});
+   * await transactionBuilder.addPostPriceUpdates(priceUpdateData);
+   * await transactionBuilder.addPriceConsumerInstructions(...)
```

Comment:
I think it would be nice to demonstrate how to access the posted price update data here 

One other comment is that users are going to wonder when to use `addPostPriceUpdates` vs `addUpdatePriceFeed` because they both seem very similar. You can clarify by adding a sentence to the comment to the effect of "Use this function if you are integrating with Pyth (FILL IN: in XXX fashion). If you are doing (FILL IN: other thing), see addUpdatePriceFeed".

And put a similar comment on addUpdatePriceFeed below also

Reply:
you could also add a comment on the class itself that explains the difference between "price updates" and "price feeds"